docs/ref/models/querysets.txt

```diff
@@ -1654,6 +1654,12 @@ than one object was found. The
 :exc:`~django.core.exceptions.MultipleObjectsReturned` exception is an
 attribute of the model class.
 
+.. note::
+	In case of no parameters, column of single-row queryset is returned.
```

Comment:
Maybe something like this? 
```
If you're filtering a queryset and expect it to return one row, you can use
``get()`` without any arguments to return the object for that row::

   Entry.objects.filter(...).get()
```
I think it can go at the bottom of this section after the last example. It doesn't need to be in a note box, I try to use those sparingly.

See if you can follow the commit message guidelines described in our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).